### PR TITLE
Update WSC task to use token aligner

### PR DESCRIPTION
**Pull request by pyeres**
This PR updates the WSC task to use the tokenization normalization and alignment routine from PR #85.

**How were these changes validated?**
These changes were validated by benchmark test: using this tokenization scheme, performance using RoBERTa-large match the [Taskmaster baseline](https://github.com/nyu-mll/jiant/pull/990#issuecomment-578271369).

### CODE REVIEW

File: jiant/tasks/lib/wsc.py [hunk 1]
```diff
@@ -71,8 +56,8 @@ def tokenize(self, tokenizer):
 class TokenizedExample(BaseTokenizedExample):
     guid: str
     tokens: List
-    span1_span: List
```
Comment (zphang) [inline jiant/tasks/lib/wsc.py:74]: Can we use the Inclusive/Exclusive span type?

**Comment by pyeres** [comment]
In issue #78 it's been pointed out that there appear to be errors in a handful of WSC test set examples. Waiting to see if/how data issues are handled before finalizing this PR.

**Comment by sleepinyourhat** [comment]
Check-in with @w4ngatang if it's not obvious how to handle some example types.

**Comment by jeswan** [comment]
@pyeres Since was determined to be an issue with test data, can this PR be closed or should these changes still be made?

**Comment by pyeres** [comment]
Hey @jeswan — yeah, I think we'll still want to make these changes. There were a couple motivations (beyond addressing issue https://github.com/jiant-dev/jiant/issues/78 which, like you said, is a data issue):
1. Dropping the dependency on `hacky_tokenization_matching`.
2. Adopting the common normalization and alignment routine established for other span alignment tasks (`normalize_tokenizations` before using `TokenAligner`, see Spr1 for example: https://github.com/jiant-dev/jiant/pull/81).

To validate changes like these we'd usually run a benchmark test, but in this case there's a chance the data we'd use to run that test will be updated (to address the issue you pointed out in https://github.com/jiant-dev/jiant/issues/78).

**Comment by pyeres** [comment]
Spoke to @jeswan offline — planning to follow up with some minor add'l changes and revalidate on benchmark.

**Comment by pyeres** [comment]
OK — benchmarked with the latest change (763a28b). RoBERTa performance on val was 72.1% (vs. 71.2% before PR changes).

**Pull request merged**